lib/ansible/plugins/action/win_copy.py

```diff
@@ -233,7 +233,7 @@ def _create_content_tempfile(self, content):
     def _create_zip_tempfile(self, files, directories):
         tmpdir = tempfile.mkdtemp()
         zip_file_path = os.path.join(tmpdir, "win_copy.zip")
-        zip_file = zipfile.ZipFile(zip_file_path, "w")
+        zip_file = zipfile.ZipFile(zip_file_path, "w", zipfile.ZIP_DEFLATED, True)
```

Comment:
Can you change this to just set `allowZip64=True` and not set the compression.

Reply:
Let me run a test here too; once that passes with your suggested setting, I will do a new commit. Stay tuned.